test/jdk/java/security/Exceptions/ChainingConstructors.java

```diff
@@ -185,6 +185,16 @@ public static void main(String[] args) throws Exception {
             throw new SecurityException("Test 16 failed");
         }
 
+        InvalidParameterException ipe =
+                new InvalidParameterException(cause);
```

Comment:
Indentation looks unusual, but it's consistent with some other parts. No need to change!